src/v2i-hub/CARMACloudPlugin/src/CARMACloudPlugin.cpp

```diff
@@ -293,7 +296,18 @@ void CARMACloudPlugin::Broadcast_TCMs()
 					PLOG(logINFO) << "Sent No ACK as Time Out: "<< sss.str() <<endl;
 					CloudSend(sss.str(),url, base_ack, method);					
 					break;

```

Comment:
Should this be continue instead of a break?  You want to continue to the next iteration, correct?